apps/api/v2/src/ee/bookings/2024-08-13/services/bookings.service.ts

```diff
@@ -997,4 +1002,52 @@ export class BookingsService_2024_08_13 {
       t: await getTranslation("en", "common"),
     });
   }
+
+  async updateBooking(bookingUid: string, body: UpdateBookingInput_2024_08_13) {
+    let bookingLocation = "";
+    const { location } = body;
+    if (!location) {
```

Comment:
Location is optional in `UpdateBookingInput_2024_08_13` so we should not throw an error if a location is missing - if we then want to allow also updating attendees this code would contain typescript errors.

Simply create a new function `updateBookingLocation` and call it if location is passed and otherwise just return existing booking from the database.